addons/payment/models/payment_provider.py

```diff
@@ -235,9 +240,9 @@ def _compute_feature_support_fields(self):
         self.update(dict.fromkeys((
             'support_express_checkout',
             'support_manual_capture',
-            'support_refund',
             'support_tokenization',
         ), None))
+        self.support_refund = 'none'
```

Comment:
This works, but you could also update the code update to have just one dict update with the appropriate value for each field, now that they don't all take the value `None`.

Reply:
Done the changes :+1: